monitoring/rules/alerts.rules

```diff
@@ -58,16 +58,23 @@ ALERT ClusterDiskLow
 
 # Zero SQL qps.
 ALERT ZeroSQLQps
-  IF sql_conns > 0 and rate(sql_query_count[5m]) == 0.0
+  IF sql_conns{job="cockroach"} > 0 and rate(sql_query_count{job="cockroach"}[5m]) == 0.0
   FOR 10m
   ANNOTATIONS {
     summary = "Instance {{ $labels.instance }} has SQL connections but no queries",
   }
 
-# Prometheus disk getting full..
```

Comment:
removed the prometheus disk full alert, this needs the prometheus node exporter running on the prometheus host, something we don't control.